src/apps/runner.js

```diff
@@ -31,6 +32,7 @@ export const runAppOp = (
     // we need to allow a 1s delay for the action to be achieved without glitch (bug in old firmware when you do things too closely)
     defer(() => delay(getEnv("MANAGER_INSTALL_DELAY"))).pipe(ignoreElements()),
     defer(() => exec(appOp, deviceInfo.targetId, app)).pipe(
+      filter((e) => "progress" in Object(e)), // Ignore the exposed allow manager events from inline install
```

Comment:
that pattern is very weird (`in Object(e)`), can you explain what Object() is doing & why is it necessary?

Reply:
`Object(e)` guarantees that whatever the event is, will be an object so I can do the `key in obj` pattern. I think I had an issue where `e` would be either `null`, or undefined, then using this makes it work since it turns it into an empty object whereas doing `"progress" in e` with `e` either `undefined` or `null` throws an uncaught typeError. I see nothing inherently wrong in this fail safe but I'm open to alternatives.